pom.xml

```diff
@@ -50,7 +50,7 @@
 		<dependency>
 			<groupId>org.springframework</groupId>
 			<artifactId>spring-web</artifactId>
-			<version>6.1.6</version>
+			<version>6.1.11</version>
```

Comment:
currently spring-web is provided by DSF 1.5.2 in version 6.1.10, might make a difference while executing unit tests.

Reply:
reset with 18d045b